homeassistant/components/volvooncall/__init__.py

```diff
@@ -164,93 +205,72 @@ def discover_vehicle(vehicle):
                         instrument.attr,
                         instrument.slug_attr,
                     ),
-                    config,
+                    self.config,
                 )
             )
 
-    async def update(now):
+    async def update(self):
         """Update status from the online service."""
-        try:
-            if not await connection.update(journal=True):
-                _LOGGER.warning("Could not query server")
-                return False
-
-            for vehicle in connection.vehicles:
-                if vehicle.vin not in data.vehicles:
-                    discover_vehicle(vehicle)
+        if not await self.connection.update(journal=True):
+            _LOGGER.warning("Could not query server")
```

Comment:
We don't want to log a warning here as it will spam the log. The coordinator will make sure to only log the first consecutive failed update at warning level.

Since we're not logging any data we can just remove this log. If you want to log data, please use debug level.

Reply:
```suggestion
```